Author: Stefan Schwarzer <sschwarzer@sschwarzer.net>
Description: Fixes path for docs.
--- a/setup.py.orig
+++ b/setup.py
@@ -29,7 +29,7 @@
   version=_version,
   packages=[_package],
   package_dir={_package: ""},
-  data_files=[("doc/ftputil", ["ftputil.txt", "ftputil.html",
+  data_files=[("share/doc/ftputil", ["ftputil.txt", "ftputil.html",
                                "README.txt", "README.html"])],
   # Metadata
   author="Stefan Schwarzer",
